.github/ISSUE_TEMPLATE/membership-request.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Membership Request
+about: Request to become a KubeEdge community member
+title: 'REQUEST: New membership for <your name>'
+labels: membership
+assignees: ''
+
+---
+
+## GitHub Username
+<!-- your github handle here -->
+
+## Organization you are requesting membership in
+<!-- The organization name here -->
+
+## Requirements
+
+- [ ] I have reviewed the [community membership guidelines]
+- [ ] I have joined in the community [mailing list] and [slack]
+- [ ] I have enabled [two-factor authentication] on my GitHub account
+- [ ] I am actively contributing to 1 or more KubeEdge subprojects
+- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
```

Comment:
Is it necessary to check  the affiliation of the account in [cncf/gitdm](https://github.com/cncf/gitdm) ?

Reply:
good idea.